docs/LightningNetwork.md

```diff
@@ -44,122 +45,146 @@ export BTCPAYGEN_LIGHTNING="implementationgoeshere"
 
-![BTCPay Server Lightning Network Setup 1](./img/lightning-node-setup/LightningNetworkNodeSetup1.jpg "BTCPay Server Lightning Network Setup 1")
+![LightningNetworkNodeSetupOverview](./img/lightning-node-setup/LightningNetworkNodeSetupOverview.jpg "BTCPay Server Lightning Network Setup Overview")
 
-![BTCPay Server Lightning Network Setup 2](./img/lightning-node-setup/LightningNetworkNodeSetup2.png "Use the internal node")
+If the internal connection fails, confirm:
+1. The Bitcoin on-chain node is fully synchronized
+2. The Internal lightning node is "Enabled" under "Lightning" > "Settings" > "BTC Lightning Settings"
 
-To ensure that your Lightning node is connected and accessible for use in your store's invoices, try to test your Lightning connection by clicking the "Public Node Info" link. Verify that a successful connection is displayed. If your internal node connection fails, make sure you have a Lightning implementation [enabled](./FAQ/LightningNetwork.md#i-previously-installed-btcpayserver-without-lightning-can-i-enable-it) on your server.
-
-If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your test connection shows as successful.
+If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your Lightning node appears "Online".
 
 ## Connecting an external Lightning Node in BTCPay
 
-If you would like to provide your own Lightning node connection string, select the "Use custom node" option. If you aren't sure of which settings to use, there are some examples provided on this page to help guide you. This is typically used to connect an external Lightning node to a BTCPay Server.
-
-![BTCPay Server Lightning Network Setup Custom](./img/lightning-node-setup/LightningNetworkNodeSetupCustom.jpg "BTCPay Server Lightning Network Setup Custom")
-
-## Getting Started with BTCPay and LND
+BTCPay Server offers the option to connect to an external Lightning node. To configure it:
+1. Go to "Lightning" > Select "Use custom node" if there is no Lightning node configured.
+2. Go to "Lightning" > Select "Settings" > Select "Change connection" > Select "Use custom node" to modify an existing connection
+3. Add the configuration details matching the lightning implementation used > "Test connection"
 
-The easiest way to use **LND implementation with BTCPay** is to use **[Ride The Lightning](https://github.com/ShahanaFarooqui/RTL)** (RTL).
 
-![Ride The Lightning in BTCPay Server](./img/RideTheLightning.png "Ride The Lightning in BTCPay Server")
+# Getting started with your LN implementation
```

Comment:
```suggestion
## Getting started with your LN implementation
```

Reply:
Given the amount of content on this particular page, I have removed the" Getting started with your LN implementation" header and promoted the implementation specific header. Let me know what you think!